### DISCUSSION TITLE
Homing Z - what to change in code to use z-end stop and not 5pin

### DISCUSSION THREAD

**Discussion by amherables** [Jyers/Marlin#256] [Q&A] [answered]
From an earlier question on Homing Z, the pre-compiled files assumes that I have the BLTouch plugged into the 5-pin port and not the Z-end stop. My question is what to change in the config file if using z-end stop and not 5pin. I believe the question was considered closed but the details on how it was fixed was not shared. Can you share the details on what to change in the code to use the z-end stop for BLTouch and not the 5-pin port? Thank you.

**Comment by Xploder** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/3930461/111068886-d9f67500-84ca-11eb-9fb8-939f71e0e43d.png)
Uncomment this line in Configuration.h

**Reply by amherables** [reply]
thank you.
